enigma-core/app/src/esgx/ocalls_u.rs

```diff
@@ -197,6 +198,32 @@ pub unsafe extern "C" fn ocall_get_deltas(db_ptr: *const RawPointer, addr: &Cont
     }
 }
 
+#[no_mangle]
+pub unsafe extern "C" fn ocall_remove_delta(db_ptr: *const RawPointer,
+                                            contract_address: &ContractAddress, _delta_index: *const u32) -> EnclaveReturn {
```

Comment:
Very minor, but I don't understand why you use a preceding underscore in name `_delta_index`.

Reply:
fixed this. (in `ocall_new_delta` as well)

Reply:
elaborated